client/src/components/ratingsReviews/ReviewsHelperFunc.jsx

```diff
@@ -14,12 +18,46 @@ const helpers = {
             return (<p>✓ I recommend this product</p>);
         }
     },
-    helpfulness: (helpNum) => {
+    
+    //checks if the review has already been clicked for helpfulness already
+    checkIfHelpfulnessClicked : (id) => {
+        if (helpfulReviewsClicked[id]) {
+            return true;
+        }
+        return false;
+    },
+    generateHelpfulness: (helpNum, id) => {
+        console.log('id', id);
         if (helpNum > 0) {
-            return (<div>
+            return (<div id="helpful">
+                <span><p id="helpful-question">Helpful? </p>
+                <p id= {"helpful-link" + id} onClick = {
+                    () => {
+                        const clickedAlready = helpers.checkIfHelpfulnessClicked(id);
```

Comment:
Naming of the variables are very clear here. Gives me a better idea of what each one does.